.golangci.yml

```diff
@@ -10,7 +10,7 @@ linters:
     - gofmt
     # - goimports
     # - golint
-    - maligned
+    - govet
```

Comment:
Oh cool! Didn't know `maligend` was replaced with govet field alignment!

Reply:
since it is just a bunch of errchecks making linter angry, should I:

* fix errcheck
or
* disable errcheck for those paths and call it a day?

Reply:
Looks like they have existed before, imo It should be fine if we call it a day and fix err check in a different PR for v4.x branch

Reply:
I think we can just never fix errcheck on v4.x and call it a day